include/utils/types/numeric_facade.h

```diff
@@ -14,7 +14,7 @@ struct NumericFacade
 {
     using value_type = T;
 
-    value_type value{};
+    value_type value{ static_cast<T>(0) };
```

Comment:
not sure if a static_cast is needed here, for simple numerical types, the default is always a zero value
```suggestion
    value_type value{ };
```

Reply:
Hem, unless I missed some improvement of the language, it is not. Basic types like `float` and `int` don't have an initial value and can contain anything. Or maybe just having the `{}` is good enough ?

Reply:
Ok, seems to be the case indeed. I thought `value_type value{};` and `value_type value;` were identical but they are not.

Reply:
https://en.cppreference.com/w/cpp/language/zero_initialization